contributing_to_docs/doc_guidelines.adoc

```diff
@@ -248,21 +240,20 @@ $ lorem.sh
 
 Without the line spaces the content is likely to be not parsed correctly.
 
-=== Inline Code or Commands
-Do NOT show full commands or command syntax inline within a sentence. See <<Command syntax and examples>> for information on how to show commands and command syntax.
+=== Command syntax and examples
+Code blocks are used to show command usage, syntax, and examples of command screen outputs or configuration files. Code blocks can be used in a few different ways:
```

Comment:
It seems weird to start both of these back to back sections with the same sentence (line 218 for "Code blocks").